ajax/libs/jquery-csv/package.json

```diff
@@ -1,20 +1,32 @@
 {
   "name": "jquery-csv",
-  "filename": "jquery.csv-0.71.min.js",
-  "version": "0.71",
-  "description": "CSV parser (complete, end-to-end, battle tested, performance optimized, 100% IETF RFC 4180 compatible) in pure js to be used with jQuery, nodejs and more",
+  "filename": "jquery.csv.min.js",
+  "version": "0.8.2",
+  "description": "A jQuery CSV parser plugin. Battle Tested | Optimized | 100% IETF RFC 4180 Complete",
   "homepage": "http://code.google.com/p/jquery-csv/",
   "keywords": [
     "jquery",
     "csv"
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/mirlord/jquery-csv"
+    "url": "https://github.com/jquery-csv/jquery-csv"
```

Comment:
@pvnr0082t 
I think the `url` is https://github.com/evanplaice/jquery-csv, and you can remove the `homepage` because it will redirect to this url